pkgs/tools/misc/pazi/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, rustPlatform}:
```

Comment:
Another nitpick, sorry :P There's a space after the opening brace but none before the closing brace.

Reply:
Done. No problem. I actually catched it for my other PR but forgot about this one.